app/controllers/publish_interface/providers/locations_controller.rb

```diff
@@ -0,0 +1,21 @@
+module PublishInterface
+  module Providers
+    class LocationsController < PublishInterfaceController
+      def index
```

Comment:
I've proposed that we authorize all actions in the PublishInterface here. Will need the following line in this action if we agree on that approach:

`authorize :provider, :index?`

Reply:
done